app/assets/stylesheets/frontend/views/_historic-appointments.scss

```
@@ -1,3 +1,15 @@
+.historic-appointments:not(.historical-past-chancellors) .featured-profiles .year-block {
+  @include media(mobile) {
+    h2.profiles {
+      margin: 0 0 $gutter-one-sixth;
```

On reflection, I agree with @kr8n3r on this:

```suggestion
      margin: 0 0 govuk-spacing(1) 0;
```